text/0000-atcompoundkey-error-path-optimization.md

```diff
@@ -0,0 +1,197 @@
+- Start Date: 2018-10-05
+- RFC PR: 
+- CppMicroServices Issue: https://github.com/CppMicroServices/CppMicroServices/issues/302
+
+# Improving the error path performance for AtCompoundkey API
+
+## Motivation
+
+Using AtCompoundKey with non-existent keys produces an exception. If the API is used extensively, the application performace is degraded as reported by a user. Users prefer not to pay for the cost of exception handling in case a compound key is not found in the AnyMap.
```

Comment:
what does extensively mean in this context? calling the API in a tight loop? Calling the API to find a key in a map twenty levels deep? all of the above?